From a12eab686c41516a51449bcc61883703393870b2 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 2 Jul 2021 19:04:56 +0200
Subject: [PATCH 104/N] [squash] Set MS_DLL_ID

This needs to be merged into "MINGW dynamic loading support"
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 78057de..bf21222 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3958,6 +3958,7 @@ then
 	*-*-mingw*)
 	DYNLOADFILE="dynload_win.o"
 	extra_machdep_objs="$extra_machdep_objs PC/dl_nt.o"
+	CFLAGS_NODIST="$CFLAGS_NODIST -DMS_DLL_ID='\"$VERSION\"'"
 	;;
 	esac
 fi
-- 
2.32.0

